assembl/static2/js/app/components/debate/common/ideaPreview.jsx

```diff
@@ -48,10 +42,16 @@ const IdeaPreview = ({
             >
               <Translate value="debate.thread.seeSubIdeas" count={numChildren} />
             </div>
+            <div>/</div>
```

Comment:
have you tried to remove this ?

Reply:
it's the visual separator between the two lines